scripts/clean-wc.sh

```diff
@@ -131,9 +131,17 @@ if [ -n "$GATE_CONS" ]; then
     "${here}/.././bin/ck8s" ops kubectl wc delete --ignore-not-found=true $GATE_CONS
 fi
 
-# Delete hnc crds
-"${here}/.././bin/ck8s" ops kubectl wc delete crds \
-    hierarchicalresourcequotas.hnc.x-k8s.io \
-    hierarchyconfigurations.hnc.x-k8s.io \
-    hncconfigurations.hnc.x-k8s.io \
-    subnamespaceanchors.hnc.x-k8s.io
+# HNC specific removal
+HNC_CRDS=$(
+    "${here}/.././bin/ck8s" ops \
+        kubectl wc api-resources \
+        --api-group=hnc.x-k8s.io \
+        -o name
+    )
+
+# Delete CRDs
+if [ -n "$HNC_CRDS" ]; then
+    # shellcheck disable=SC2086
+    # We definitely want word splitting here.
+    "${here}/.././bin/ck8s" ops kubectl wc delete crds $HNC_CRDS
+fi
```

Comment:
```suggestion
# HNC specific removal
read -ra HNC_CRDS <<< $(
    "${here}/.././bin/ck8s" ops \
        kubectl wc api-resources \
        --api-group=hnc.x-k8s.io \
        -o name
    )

# Delete CRDs
if [ -n "$HNC_CRDS" ]; then
    "${here}/.././bin/ck8s" ops kubectl wc delete crds "${HNC_CRDS[@]}"
fi
```

Reply:
Is this to fix the shellcheck warning? Or why would we want this change?

Reply:
Something about the word splitting?

Reply:
Yea this way you can do word spitting while still keeping the linter happy. I did not mean to change anything functionally so if I did please let me know.

While highly unlikely and not an critical risk here, even if you want word splitting you could still cause globbing without keeping the double quotes so I thought why not keep it.

Reply:
You could even change the conditional to `[ ${#HNC_CRDS[@]} -ne 0 ]`

Reply:
Another possibility, hopefully decently clear and readable:

```suggestion
# HNC specific removal
"${here}/.././bin/ck8s" ops \
        kubectl wc api-resources \
        --api-group=hnc.x-k8s.io \
        -o name \
  | xargs --no-run-if-empty \
    "${here}/.././bin/ck8s" ops kubectl wc delete crds $HNC_CRDS
```

Reply:
Hmm :thinking: I think that I'd lean a bit more to use the array instead of xargs since that's more in line with how we do it elsewhere. But I like that approach as well.